app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -8,7 +8,7 @@ class ConfigReportsController < V2::BaseController
       before_action :setup_search_options, :only => [:index, :last]
       before_action :compatibility, :only => :create
 
-      add_smart_proxy_filters :create, :features => proc { ConfigReportImporter.authorized_smart_proxy_features }
+      add_smart_proxy_filters :facts, :features => proc { Foreman::Plugin.fact_importer_registry.fact_features }
```

Comment:
the endpoint here is called `:create`, not `:facts`. Also i'm not sure if there are plugins that import reports but don't have a fact importer, especially now that all fact importers are in core?

Reply:
Oh copy and paste error. Good find.

I do not understand why this endpoint is used, normally /v2/host/xxx/facts should be used. But it should save the same purpose I guess and the change I refer to should should change both.